slash/react/src/Form/File/constants.tsx

```diff
@@ -0,0 +1,4 @@
+export enum FileActions {
+  add = "add",
+  delete = "delete",
+}
```

Comment:
Pour ce genre de chose il est plutot recommandé de passer par un union type :

```suggestion
export type FileActions ="add" | "delete";
```

La raison c'est principalement que le type disparait au runtime, contrairement a l'enum. et que ca ne force pas l'export du type pour etre utilisé